javascript/javascript1/week4/getReplay.js

```diff
@@ -0,0 +1,74 @@
+let toDo = [];
+const monthNames = [
+  "January",
+  "February",
+  "March",
+  "April",
+  "May",
+  "June",
+  "July",
+  "August",
+  "September",
+  "October",
+  "November",
+  "December",
+];
+function getReply(command) {
+  switch (command) {
+    case "Hello my name is Maher":
+      return "Nice to meet you Maher";
+
+    case "What is my name?":
+      return "Your name is Maher";
+
+    case "Add fishing to my todo":
+      toDo.push("fishing");
+      return "fishing added to your todo";
+
+    case "Add singing in the shower to my todo":
+      toDo.push("singing in the shower");
+      return "singing added to your todo";
+
+    case "Remove fishing from my todo":
+      const index = toDo.indexOf("fishing");
+      if (index > -1) {
+        toDo.splice(index, 1);
+      }
+      return "Removed fishing from your todo";
+
+    case "What is on my todo?":
+      return "you have " + toDo.length + " todos " + '"' + toDo + '"';
+
+    case "What day is it today?":
+      return (
+        new Date().getDate() +
+        " of " +
+        monthNames[new Date().getMonth()] +
+        " " +
+        new Date().getFullYear()
+      );
```

Comment:
There is a shorter way available at modern JavaScript like this
```js
  if (command === 'what day is it today?') {
    return new Date().toLocaleDateString('en-US', {
      day: 'numeric',
      year: 'numeric',
      month: 'long'
    });
  }
```

Reply:
Hi Zuhair. in my solution I traied to show the date in form (dd-mm-yyyy).